includes/admin/class-wc-admin-settings.php

```diff
@@ -249,7 +251,9 @@ public static function output_fields( $options ) {
 							echo '<h2>' . esc_html( $value['title'] ) . '</h2>';
 						}
 						if ( ! empty( $value['desc'] ) ) {
+							echo '<div id="' . esc_attr( sanitize_title( $value['id'] ) ) . '-description">';
```

Comment:
I see a couple lines down on line 259 `if ( ! empty( $value['id'] ) )` which implies the ID may not be set. Do we need a check here to handle that or will it definitely always be present?

Reply:
It's set further up to empty string if missing.